modules/TypeDefinitions.js

```diff
@@ -28,6 +28,13 @@ export type RouteProps = {
 }
 
 /**
+ * Style
+ */
+
+export type Styles = {[key: string]: Object};
+export type StyleSheet = Styles | Array<Styles>;
```

Comment:
Flow wasn't recognizing the `StyleSheet` type from React Native. But do you mean that I should use : 

```
type Atom = number | bool | Object | Array<?Atom>;
export type StyleObj = Atom;
```

instead of what I've written?